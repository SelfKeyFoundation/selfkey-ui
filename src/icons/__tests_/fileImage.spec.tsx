import * as React from 'react';
import { create } from 'react-test-renderer';
import FileImageIcon from '../file-image';

describe('Filed Image Icon', () => {
  it('should show Filed Image Icon', () => {
    const component = create(
      <FileImageIcon />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});