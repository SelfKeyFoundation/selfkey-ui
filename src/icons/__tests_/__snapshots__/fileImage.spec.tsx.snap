// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filed Image Icon should show Filed Image Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root-1 Component-root-0-1-1"
  focusable="false"
  role="presentation"
  viewBox="0 0 29 36"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="ðŸ’»-ID-Dashboard-Image-Icon"
    stroke="none"
    strokeWidth="1"
  >
    <g
      fill="#697C95"
      fillRule="nonzero"
      id="File-Image-Icon"
      transform="translate(-392.000000, -1475.000000)"
    >
      <g
        id="Area-box-3"
        transform="translate(150.000000, 1262.000000)"
      >
        <g
          id="Table"
          transform="translate(14.000000, 80.000000)"
        >
          <g
            id="Col2"
            transform="translate(228.000000, 11.000000)"
          >
            <g
              id="line"
              transform="translate(0.000000, 120.000000)"
            >
              <path
                d="M0,2 L0,38 L28.173913,38 L28.173913,37.2173913 L28.173913,12.4979616 L17.538383,2 L0,2 Z M1.56521739,3.56521739 L16.4347826,3.56521739 L16.4347826,13.7391304 L26.6086957,13.7391304 L26.6086957,36.4347826 L1.56521739,36.4347826 L1.56521739,3.56521739 Z M18,4.65506112 L25.6166781,12.173913 L18,12.173913 L18,4.65506112 Z M20.7391304,17.130944 C20.346955,17.13101 19.9708769,17.2822817 19.6936656,17.5514663 C19.4164543,17.8206509 19.2608285,18.1856874 19.2610393,18.5662366 C19.2614707,19.3581076 19.9230668,19.9998626 20.7391304,20 C21.5551941,19.9998626 22.2167902,19.3581076 22.2172216,18.5662366 C22.2174323,18.1856874 22.0618066,17.8206509 21.7845953,17.5514663 C21.5073839,17.2822817 21.1313059,17.13101 20.7391304,17.130944 Z M11.0879757,19.2173913 C10.8290617,19.2070522 10.5818126,19.3255258 10.4276497,19.5337977 L4.5886547,27.4256112 C4.41336043,27.6632684 4.3867541,27.9793567 4.51985435,28.2429725 C4.6529546,28.5065884 4.92309883,28.6728467 5.21841052,28.6728938 L16.7680025,28.6728938 C16.7858304,28.6735035 16.8036738,28.6735035 16.8215017,28.6728938 C16.8245585,28.6729117 16.8276154,28.6729117 16.8306723,28.6728938 L21.9237433,28.6086957 C22.2236684,28.6044051 22.4947607,28.4290816 22.6216968,28.1573083 C22.7486329,27.885535 22.7090828,27.565121 22.5198712,27.3323712 L17.4283286,21.0837299 C17.277141,20.8987033 17.0497168,20.792872 16.8108018,20.7963654 C16.5641989,20.799853 16.3336643,20.9193686 16.188689,21.1188863 L14.5210602,23.4162701 L11.688689,19.538383 C11.5479428,19.3453038 11.326712,19.227089 11.0879757,19.2173913 Z M11.0528194,21.3221806 L13.8943612,25.212296 C13.8978767,25.2169211 13.9014435,25.221507 13.905061,25.2260527 L15.2792123,27.1076764 L6.77139965,27.1076764 L11.0528194,21.3221806 Z M16.8566572,22.861413 L20.2836279,27.0648779 L17.2158629,27.1046196 L15.4916781,24.7430368 L16.8566572,22.861413 Z"
                id="icon-file-image"
              />
            </g>
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
